src/commands/init.ts

```diff
@@ -141,10 +95,77 @@ export class InitCommand implements Command {
           };
           inquirer.prompt([prompt]).then((answers) => {
             let generatorName = answers.generatorName;
-            runGenerator(generatorName, getDisplayName(generatorName));
+            runGenerator(generatorName, InitCommand.getDisplayName(generatorName));
           });
         }
       });
     });
   }
+
+  private static registerDefaultGenerators(env: YeomanEnvironment): void {
```

Comment:
I don't think these need to be private static methods, they can just be unexported functions in this module.